### ISSUE TITLE
problem with the installation on my Mac

### ISSUE THREAD

**Issue by melanileo** [kaixxx/noScribe#55]
### Discussed in https://github.com/kaixxx/noScribe/discussions/54

<div type='discussions-op-text'>

<sup>Originally posted by **melanileo** April 30, 2024</sup>
I have issues downloading the program on my MacBook. its says it cannot be open, since Mac can't search for bugges.I really need this program for my transcription.</div>

**Comment by gernophil** [comment]
If you get this message it seems to be a gatekeeper issue and not a macOS version issue. Did you follow the install manual for intel?

**Comment by melanileo** [comment]
Hello!

I tried to install no scrib on my Mac, just as it is described in the instructions (Intel). Unfortunately, it does not work: I cannot open the program. I then downloaded the Apple Silicon version and didn't work either.
When I click on the program, it looks like it is going on for a moment, but then nothing happens and it doesn't appear in the dock either.

I have the MacIs Big Sur 11.7.10, Intel Iris Pro 1536 MB 2,5 GHz Quad core intel core i7
